.env.sample

```diff
@@ -36,4 +36,4 @@ DJANGO_TIMEZONE=America/Detroit
 #OIDC_OP_AUTHORIZATION_ENDPOINT
 #OIDC_OP_TOKEN_ENDPOINT
 #OIDC_OP_USER_ENDPOINT
-#OIDC_OP_JWKS_ENDPOINT
```

Comment:
Since this change is not related to this fix and deleting the space, I feel if this change should not be reflected in the PR.

if this is redundant then remove it